webserver/views/api/v1/core.py

```diff
@@ -274,13 +312,18 @@ def get_many_highlevel():
 
        {"mbid1": {"offset1": {document},
                   "offset2": {document}},
-        "mbid2": {"offset1": {document}}
+        "mbid2": {"offset1": {document}},
+        "mbid_mapping": {"MBID1": "mbid1"}
```

Comment:
This is the core change, now we'll return this `"mbid_mapping"` field with bulk queries. I decided to _always_ include the key, but only include items here which weren't in the normalised format. So if a valid, lower-case mbid is passed in, nothing will show here. Also, if the mbid doesn't exist in the database, it'll appear here, but won't be in the rest of the dictionary. I think this is a good trade-off, but could be convinced to always put all parameters in this mapping if we think it's needed.

Reply:
I'm happy to keep it as it is now.